ingestion/src/metadata/ingestion/source/database/postgres/query_parser.py

```diff
@@ -70,8 +73,39 @@ def get_sql_statement(self, *_) -> str:
         return self.sql_stmt.format(
             result_limit=self.config.sourceConfig.config.resultLimit,
             filters=self.filters,
+            time_column_name=self.get_postgres_time_column_name(),
         )
 
+    def get_postgres_version(self) -> Optional[str]:
+        """
+        return the postgres version in major.minor.patch format
+        """
+        query = "show server_version"
+        try:
+            results = self.engine.execute(query)
+            for res in results:
+                version_string = str(res[0])
+                opening_parenthesis_index = version_string.find("(")
+                if opening_parenthesis_index != -1:
+                    return version_string[:opening_parenthesis_index].strip()
+                return version_string
+        except Exception as err:
+            logger.warning(f"Unable to fetch the Postgres Version - {err}")
+            logger.debug(traceback.format_exc())
+        return None
```

Comment:
That won't be necessary [since](https://github.com/open-metadata/OpenMetadata/pull/11536/files#diff-a1aff1fb398b638247382c74ed1afeb32bab60ab7ea9e0bcaf55dce338305ad1R101) here the default name is being handled